lib/dec/tail.c

```diff
@@ -29,11 +29,24 @@
 			break;
 		}
 		if ((q = base64_table_dec[*c++]) >= 254) {
-			st.eof = 1;
-			// When q == 254, the input char is '='. Return 1 and EOF.
-			// Technically, should check if next byte is also '=', but never mind.
-			// When q == 255, the input char is invalid. Return 0 and EOF.
-			ret = (q == 254) ? 1 : 0;
+			// When q == 254, the input char is '='.
+			// Check if next byte is also '=':
+			if (q == 254) {
+				if (srclen-- != 0) {
```

Comment:
Nope, I checked that. That's why there's the new flag `aeof` in case of streaming mode.